pod/perldelta.pod

```diff
@@ -315,9 +315,11 @@ XXX List any platforms that this version of perl no longer compiles on.
 
 =over 4
 
-=item XXX-some-platform
+=item NetWare
 
-XXX
+Support code for Novell NetWare has been removed.  NetWare was a
+server operating system by Novell.  The port was last updated in July
+2002, and the platform itself in May 2009.
 
```

Comment:
I realise after hitting "finish" that, *if* the other comment means an `--amend` is needed, then maybe also tweak this text to add that unrelated changes accidentally broke the build for the existing port in Sept 2009, and in 12 years no-one has reported this.

(Otherwise we tweak the text in its own commit)